code/datums/components/aiming.dm

```diff
@@ -192,16 +190,11 @@ AIMING_DROP_WEAPON means they selected the "drop your weapon" command
 			fire()
 			return
 		if(RAISE_HANDS)
-			alert_message = "raise your hands!"
-			alert_message_3p = "raise their hands!"
+			user.say(pick(list("Put your hands above your head!", "Hands! Now!", "Hands up!")), forced = "Weapon aiming")
```

Comment:
Byond supports pick from parameters, a list input isn't required
![image](https://user-images.githubusercontent.com/26465327/176997260-14e3c393-5817-458c-aa11-277ebd60e28c.png)

Reply:
Removed the lists.